Casamari/MNC009.xml

```diff
@@ -424,16 +426,13 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                
                <history>
                   <origin>
-                     <origDate notBefore="1882" notAfter="1889" evidence="prosopography">
-                        Metropolitan <persName ref="PRS7830Petros">Ṗeṭros</persName>, who was in charge from <date>1882</date>, is mentioned on <locus target="#23r #37v"/>. 
-                        King <persName ref="PRS10378Yohanne">Yoḥannǝs IV</persName>, who reigned from <date>1872</date> to 
-                        <date>1889</date>, is mentioned on <locus target="#24v"/>. This indicates that the manuscript was written between <date>1882</date> and <date>1889</date>.
-                     </origDate>
-                  </origin>
-                  <provenance>
-                     The name of <persName ref="PRS12962WaldaMa">Walda Māryām </persName> is frequently mentioned in the supplication formulas throughout the manuscript. 
+                     <origDate notBefore="1882" notAfter="1889" evidence="prosopography"/>
+                     Metropolitan <persName ref="PRS7830Petros">Ṗeṭros</persName>, who was in charge from <date>1882</date>, is mentioned on <locus target="#23r #37v"/>. 
```

Comment:
who was in charge from `<date>1882</date>`
 the second year "to xxx" is missing, isn't?